### PR TITLE
Set up humble instead of main

### PR DESCRIPTION
**Pull request by giusebar**
The default branch should be humble, now that we introduced jazzy

### CODE REVIEW

File: .github/workflows/monthly.yaml [hunk 1]
```diff
@@ -5,7 +5,11 @@ on:
...
-    uses: ubuntu-robotics/ros2-teleop-snap/.github/workflows/snap.yaml@main
+  humble:
+    uses: ubuntu-robotics/ros2-teleop-snap/.github/workflows/snap.yaml@humble
     with:
-      branch-name: "main"
+      branch-name: "humble"
+  jazzy:
+    uses: ubuntu-robotics/ros2-teleop-snap/.github/workflows/snap.yaml@jazzy
+    with:
+      branch-name: "jazzy"
```
Comment (artivis) [inline .github/workflows/monthly.yaml]: This should be in a different pr nop?

**Review by artivis — APPROVED** [review]

**Review by Guillaumebeuzeboc — APPROVED** [review]
